src/components/RecentSection/RecentSection.tsx

```diff
@@ -0,0 +1,39 @@
+import { IPublication } from "@/utils/interfaces";
+import Carousel from "../Carousel/Carousel";
+import {
+  Button,
+  ContentWrapper,
+  HeaderWrapper,
+  RightIcon,
+  Subtitle,
+  Title,
+  Wrapper,
+} from "./RecentSection.styles";
+import { Header } from "./RecentSection.styles";
+
+export const RecentSection = ({
+  header,
+  content,
+}: {
+  header: { fields: any };
+  content: { fields: IPublication }[];
+}) => {
+  const { id, title, subtitle } = header.fields;
+
+  return (
+    <Wrapper id={id}>
+      <HeaderWrapper>
+        <Header>
+          <Title href="#new">
+            {title} <RightIcon id="expand" size={14} />
+          </Title>
+          <Button href="#new">Ver todos</Button>
```

Comment:
```suggestion
            {/* <Button href="#new">Ver todos</Button> */}
```
We can comment this since we don't have this page implemented yet